config/config.go

```diff
@@ -583,7 +583,7 @@ func hasRootPrivilege() bool {
 }
 
 // TableLockEnabled uses to check whether enabled the table lock feature.
-func TableLockEnabled() bool {
+var TableLockEnabled = func() bool {
```

Comment:
I don't think an exported global variable is an elegant solution. Is there any risk to be assigned an undesired value?

Reply:
Make sense. Any better solution?